go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/open-networks/go-msgraph
+module github.com/mypricehealth/go-msgraph
```

Comment:
Please revert this change from a2b6df674e14ce4866ae030f2088992040afd056 to `module github.com/open-networks/go-msgraph`